### ISSUE TITLE
What is the Status of Many to many through associations?

### ISSUE THREAD

**Issue by gaurav21r** [balderdashy/waterline#1220]
The docs here https://github.com/balderdashy/waterline-docs/blob/master/models/associations/many-to-many-through.md say coming soon. But its been a while and I suspected that this has been implemented but the docs hasn't been updated.

It would be great if @particlebanana @devinivy or someone could update status on the same. If this done, I'd be more than happy to send a PR to the doc repo to add some relevant documentation :+1: 

As the doc already highlights this introduces a very much needed real world use case where we need to assign attributes to the join table.

**Comment by particlebanana** [comment]
@gaurav21r see https://github.com/balderdashy/waterline/issues/1133#issuecomment-158526482

Basically the current state is that you can define custom tables to use as a join table. The populated results from either of the many sides will not include any extra fields on the join table though.

This would require changes to waterline-cursor. If you would like to update the Roadmap with more info on what is requested that would be great! https://github.com/balderdashy/waterline/blob/master/ROADMAP.md

**Comment by sailsbot** [comment]
Thanks for posting, @gaurav21r.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
